api/base/filters.py

```diff
@@ -29,15 +31,49 @@ def query_params_to_fields(query_params):
 def intersect(x, y):
     return x & y
 
+
 class ODMFilterMixin(object):
     """View mixin that adds a get_query_from_request method which converts query params
     of the form `filter[field_name]=value` into an ODM Query object.
 
     Subclasses must define `get_default_odm_query()`.
+
+    Serializers that want to restrict which fields are used for filtering need to have a variable called
+    filterable_fields which is a list of strings representing the field names as they appear in the serialization.
     """
+
+    # TODO Handle simple and complex non-standard fields
+
     TRUTHY = set(['true', 'True', 1, '1'])
     FALSY = set(['false', 'False', 0, '0'])
 
+    # For the field_comparison_operators, instances can be a class or a tuple of classes
+    field_comparison_operators = [
+        {
+            'instances': ser.CharField,
```

Comment:
Happy to use whatever. I only used instances because it's being passed into the isinstance function.